install/README.md

```diff
@@ -0,0 +1,23 @@
+# Quick install scripts
+
+This directory contains scripts for installing Babelfish for PostgreSQL. 
+
+By running the install.sh script it will perform the installation process for your linux distribution. 
+
+Currently the supported linux distribution for this script are:
```

Comment:
How about:

This directory contains scripts that install Babelfish for PostgreSQL.  Select the install.sh script for your Linux platform to install Babelfish on your system. 

Currently the supported Linux distributions are: